### DISCUSSION TITLE
I am a student but I cannot apply for student certification

### DISCUSSION THREAD

**Discussion by whwangovo** [community/community#70132] [GitHub Education]
### Hi everyone,

I am a student from China and am trying to apply for student certification. But I am getting the following tips:
![image](https://github.com/community/community/assets/79965746/80748797-20de-4952-9317-b1df1b1f912c)
However, I tried to select the "OTHER" button in the picture, but there is no place to select it in the page. 
Meanwhile, I want to change the first name of my account, but he doesn't accept Chinese. But if I use English it will show a mismatch.
I am indeed a student so can someone please come and help me?T_T

**Comment by welmarstone** [comment]
It seems to be a nationality-specific problem, so I'd recommend you to contact their support team. I don't think they answer that late  mate.

**Comment by whwangovo** [comment]
Hello, I have received your email.
